### ISSUE TITLE
Keep a log of all sent emails

### ISSUE THREAD

**Issue by zwouter** [Inter-Actief/amelie#625]
If an email is sent using the website, for example through the members' administration or activity enrolments, there is no way to retrieve this email.  To be able to check what exactly was sent, I believe it would be very nice if all of these emails were logged somewhere.

An example of where this would be useful, I sent out a GMM invite last week, in which the starting times of the GMM were wrong in the Durch version. Since I have my language preferences in Dutch, I could not check whether these times were correct in English. Therefore, I had to do quite some research before I knew whether I should send the rectification email to members with English preferences as well. With some sort of logs, my day would have been a bit better :)

**Comment by zwouter** [comment]
I don't think there will be privacy issues if only board members have access to such logs.

**Comment by WKobes** [comment]
Logging sent e-mails means that sometimes e-mails are stored that contain personal information. That has privacy consequences irregardless of who can access it. Should at least be added to the privacy regulations: privacy.ia.utwente.nl

**Comment by zwouter** [comment]
Fair point, in my opinion, the emails can also be logged with the template names (so {{recipient.first_name}} instead of the personal information). I don't think the personal information would even be relevant to store.
